routes/accessControl.js

```diff
@@ -0,0 +1,25 @@
+var tokenHelper = require('../util/shared/tokenHelper.js');
+var util = require('../util/shared/util.js');
+
+
+module.exports = function(app){
```

Comment:
[blocker]This is definitely wrong here: 

1. Controller should not do any routing chain changes. Move this logic to routes module.
2. You should not apply any route specific info to app directly. You should apply it to rootRouter instead. This code here breaks the contextRoot feature.